brainstormforce/ultimate-addons-for-gutenberg
# Fix : Responsive conditions appear at the inappropriate position in other blocks

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/999e91fd-4a41-4ad0-ab8e-0e33a57b6554?redirect=true&view=6d3bcc3d-b047-4739-8e05-d9621e9f9445&t=attributes

### Screenshots

- https://share.getcloudapp.com/7KuQPArP

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop core/gallery block add an image.
- And check our extension tab should be visible at last above the advanced tab.
- Check with other blocks too.
- For Responsive condition - Show which block is hidden in the editor the same as elementor. ( Test cases added in card )
- [Screencast](https://share.getcloudapp.com/p9u6k62q) (different condition to diff block)

### Checklist:
